### DISCUSSION TITLE
how do i use the chat commands?

### DISCUSSION THREAD

**Discussion by omdtaoq** [winstxnhdw/lc-hax#60] [Q&A] [answered]
whenever I test solo, they do nothing and appear as normal chat messages. Am I doing something wrong or does it only work in multiplayer?
![image](https://github.com/winstxnhdw/lc-hax/assets/154767166/daa7b0a2-5bc3-4ce9-a88a-e97722f6aba9)
Can it also possibly be due to my username on steam being an issue?

**Comment by omdtaoq** [comment]
> whenever I test solo, they do nothing and appear as normal chat messages. Am I doing something wrong or does it only work in multiplayer? ![image](https://private-user-images.githubusercontent.com/154767166/292693908-daa7b0a2-5bc3-4ce9-a88a-e97722f6aba9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM0NjUwMzYsIm5iZiI6MTcwMzQ2NDczNiwicGF0aCI6Ii8xNTQ3NjcxNjYvMjkyNjkzOTA4LWRhYTdiMGEyLTViYzMtNGNlOS1hODhhLWU5NzcyMmY2YWJhOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyNVQwMDM4NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZjhjYTY5YjljNDg5NDZlNWEwMDJmYmRkNDBiMDlkMWEwODBmNWRlYWU5ZjYxZDVlMTdlMjgwZTdiZGU2YWVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dQGdXHyPnFSaXq8nDgZTP9Jjo2y6H65ZxXhsihKjw-Y) Can it also possibly be due to my username on steam being an issue?
The username being the issue is unlikely, as I changed it and still nothing happened. 
![image](https://github.com/winstxnhdw/lc-hax/assets/154767166/dfa92428-9800-43ff-a991-7b68d91ebc55)
Tried two commands, which did not work.

**Comment by winstxnhdw** [comment]
You have not even injected the cheats, my dude.

**Reply by omdtaoq** [reply]
I might not be doing it right cause the binded commands work (the m3 ones)
Do you open up launch.bat and then leave it open? (after running requirements.bat)

**Reply by omdtaoq** [reply]
Some of the features of the program are retained, like possession and scanning through walls, but the only non functional thing are the chat commands. It's unlikely that my mods affect any of the functions since they're all cosmetic apart from more company and the one that adds fake fire exits.
![image](https://github.com/winstxnhdw/lc-hax/assets/154767166/1eae938e-8984-48a9-a063-d3762c45f4a9)

https://github.com/winstxnhdw/lc-hax/assets/154767166/6fcef9a5-44c9-45d7-9cbd-5de45d98997d

**Comment by omdtaoq** [comment, marked as answer]
Seems like there were some mods using "/" as their prefix for their own commands and it caused a conflict, lmao.

**Reply by winstxnhdw** [reply]
Great way to know if the cheats are working is when the command prints your name as USER instead of your Steam username.

In the meantime, I can try forcing priority for my commands so that mods won't interfere with it in the future.

You do not have to keep `launch.bat` up by the way, but it can be convenient to do so if you restart your game regularly.

**Reply by winstxnhdw** [reply]
Nevermind, rightfully, because you are injecting at runtime, my code should already be given most priority. This means whatever mod you are doing is overriding the chat in a more dubious way than I initially thought.

**Reply by omdtaoq** [reply]
If you are wondering about which mods caused this, it's the boombox controller mod by KoderTeh [linked here](https://thunderstore.io/c/lethal-company/p/KoderTeh/Boombox_Controller/)
There was also one for the tv but it got taken down for some reason

**Reply by winstxnhdw** [reply]
Ah nothing better than mods that implement more features than it has to.

**Reply by omdtaoq** [reply]
oh btw are the chat commands visible for other players, do they see the "user: (insert message)" stuff? i've been playing around with it and maybe if i wanna do some trolling in multiplayer i wouldnt want them to know

**Reply by winstxnhdw** [reply]
It's not visible.
